cmd/fleet/cron.go

```diff
@@ -898,7 +898,8 @@ func trySendStatistics(ctx context.Context, ds fleet.Datastore, frequency time.D
 	if err != nil {
 		return err
 	}
-	if !ac.ServerSettings.EnableAnalytics {
+
+	if !ac.ServerSettings.EnableAnalytics && !license.IsPremium(ctx) {
```

Comment:
Let's add a comment about this for future readers (`// when the license is premium we always send statistics, etc.`).